### ISSUE TITLE
python3.6-numpy does not exist

### ISSUE THREAD

**Issue by bocekm** [Chadsr/NordVPN-NetworkManager#89]
Package `python3.6-numpy` does not exist, `python3-numpy` does.
```
Problem: cannot install the best update candidate for package nordnm-0.4.0-1.noarch
  - nothing provides python3.6-numpy >= 1.12.0 needed by nordnm-0.5.1-1.noarch
```

**Comment by Chadsr** [comment]
Thanks for pointing this out. I will get it patched ASAP. The last release was admittedly a little rushed and not tested as thoroughly as it should have been...

Somehow I also managed to remove my package installation tests on Travis and it has only just came to my attention because of this. I will have those put back in along with the fix.

**Comment by Chadsr** [comment]
This has been fixed in the RPM package for v0.5.4.  I also created a new issue regarding the creation of more cross distribtion package tests, so that any little issues can be found and fixed before a problematic package gets released (#94).

Since the topic of this issue should no longer be a problem, I will close it now. Thanks!
